src/managers/GuildBanManager.js

```diff
@@ -135,7 +135,7 @@ class GuildBanManager extends CachedManager {
    * @example
    * // Ban a user by id (or with a user/guild member object)
    * guild.bans.create('84484653687267328')
-   *   .then(user => console.log(`Banned ${user.username ?? user.id ?? user} from ${guild.name}`))
+   *   .then(user => console.log(`Banned ${user.user?.tag ?? user.tag ?? user} from ${guild.name}`))
```

Comment:
This looks completely weird.. user.user?
I know it can be a member, or a user..or an id (why did we do this again), but maybe renaming the result variable would be better? @iCrawl thoughts?